src/npcmove.cpp

```diff
@@ -1054,44 +1048,41 @@ void npc::move_to(int x, int y)
             recoil = int(recoil / 2);
         }
     }
+
     if (has_effect("stunned")) {
-        x = rng(posx() - 1, posx() + 1);
-        y = rng(posy() - 1, posy() + 1);
+        p.x = rng(posx() - 1, posx() + 1);
+        p.y = rng(posy() - 1, posy() + 1);
+        p.z = posz();
     }
-    if (rl_dist(posx(), posy(), x, y) > 1) {
-        /*
-          debugmsg("Tried to move_to more than one space! (%d, %d) to (%d, %d)",
-                   posx, posy, x, y);
-          debugmsg("Route is size %d.", path.size());
-        */
-        int linet;
-        std::vector<point> newpath;
-        if (g->m.sees(posx(), posy(), x, y, -1, linet)) {
-            newpath = line_to(posx(), posy(), x, y, linet);
+
+    if( rl_dist( pos3(), p ) > 1) {
+        int linet1, linet2;
+        std::vector<tripoint> newpath;
+        if( g->m.sees( pos3(), p, -1, linet1, linet2 ) ) {
+            newpath = line_to( pos3(), p, linet1, linet2 );
         } else {
-            newpath = line_to(posx(), posy(), x, y, 0);
+            newpath = line_to( pos3(), p, 0, 0 );
```

Comment:
Same simplification with `map::sees` setting `linet` to 0 can be done here, can it not?